src/createAnimatedComponent.tsx

```diff
@@ -602,14 +604,18 @@ export default function createAnimatedComponent(
       this._attachInlineProps();
     }
 
-    _setComponentRef = setAndForwardRef<Component>({
+    _setComponentRef = setAndForwardRef<Component | HTMLElement>({
       getForwardedRef: () =>
         this.props.forwardedRef as MutableRefObject<
           Component<Record<string, unknown>, Record<string, unknown>, unknown>
         >,
       setLocalRef: (ref) => {
-        // TODO update config
-        const tag = findNodeHandle(ref);
+        // TODO update config)
```

Comment:
```suggestion
        // TODO update config
```

Reply:
Done in [01d5140](https://github.com/software-mansion/react-native-reanimated/pull/4866/commits/01d5140f66d673675c05b685a311bf18fd8b5293), though I'm not sure whether provided explanation is sufficient. Could you have a look at it?